<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicSlider</class>
 <widget class="QWidget" name="musicSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Slider_box" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Cur_time">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font-size:12px;</string>
        </property>
        <property name="text">
         <string>1:32</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Line_box" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QFrame" name="In_line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>8</y>
           <width>560</width>
           <height>4</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;  </string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="Out_line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>8</y>
           <width>200</width>
           <height>4</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(0, 255, 0);
border-radius: 2px;  </string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="End_time">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font-size:12px;</string>
        </property>
        <property name="text">
         <string>3:36</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
